tox.ini

```diff
@@ -4,6 +4,7 @@ envlist = flake8-py3, py26, py27, py32, py33, py34, pypy
 [testenv]
 deps= -r{toxinidir}/dev-requirements.txt
 commands=
+    pip install .[socks]
```

Comment:
Do we want it? It'll cause PyOpenSSL et al to get installed here, which may not be all that sensible a thing to do.